src/redux&saga/actions/cardsActions.ts

```diff
@@ -0,0 +1,14 @@
+export const getCards = (payload: any) => ({
+  type: 'GET_CARDS',
```

Comment:
try to save all constant separately

example: 
`export const GET_CARDS = 'GET_CARDS'`
`export const REDEFINE_CARD_LIST = 'REDEFINE_CARD_LIST'`
...
`import GET_CARDS from './..'`

`export const cardsReducer = (state,  action) => {`

`switch (action.type) {`

  `  case GET_CARDS':`
      `return { ...state };`
`}`